scripts/subscriptions.json

```diff
@@ -0,0 +1,5 @@
+[
```

Comment:
this shouldn't be checked in or if you want to have one as an example should have `.example` in it with the actual one in the gitignore